app/models/transaction.rb

```diff
@@ -40,7 +41,7 @@ def total
   delegate :currency, to: :bank_account, allow_nil: true
   delegate :income?, :expense?, to: :category, allow_nil: true
 
-  default_scope { order(created_at: :desc) }
+  scope :ordered, -> { order('created_at DESC') }
```

Comment:
@yunixon looks like you will not need this line after changing ransack ordering.